obspy/io/stationtxt/core.py

```diff
@@ -248,10 +251,136 @@ def _read(obj):
         inv.networks.extend(list(networks.values()))
     else:
         # Cannot really happen - just a safety measure.
-        raise NotImplementedError("Unkown level: " + str(level))
+        raise NotImplementedError("Unknown level: " + str(level))
     return inv
 
 
+def _format_time(value):
+    if isinstance(value, UTCDateTime):
+        if value.microsecond == 0:
+            return value.strftime("%Y-%m-%dT%H:%M:%S")
+        else:
+            return value.strftime("%Y-%m-%dT%H:%M:%S.%f")
```

Comment:
Perhaps the ObsPy generation of this format could include a 'Z' specifier as an option?  As @nick-iris says we need a version sans-Z to use in an fdsnws-station implementation (which, regrettably, has no Z).